### PR TITLE
Feature: logged user can see all topics

### PR DESCRIPTION
**Pull request by ivanfelfo**
Feature:
- logged user can see all topics

### CODE REVIEW

File: app/controllers/v1/topics_controller.rb [hunk 1]
```diff
@@ -0,0 +1,10 @@
+module V1
+  class TopicsController < ApplicationController
+    before_action :authenticate_v1_user!
+
+    def index
+      @topics = Topic.all
```
Comment (martinjaimem) [inline app/controllers/v1/topics_controller.rb:6]: What happens if they are `100`, `1000`, `100000`?

What you can do instead is to paginate the topics: [you can use this gem for that](https://ddnexus.github.io/pagy/how-to#quick-start&gsc.tab=0)

File: app/controllers/v1/topics_controller.rb [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+      render json: { topics: @topics }
```
Comment (martinjaimem) [inline app/controllers/v1/topics_controller.rb:7]: what about using `jbuilder`?

File: app/controllers/v1/topics_controller.rb (hunk 1 above)
Comment (ivanfelfo) [reply app/controllers/v1/topics_controller.rb]: 👍

File: app/controllers/v1/topics_controller.rb (hunk 2 above)
Comment (ivanfelfo) [reply app/controllers/v1/topics_controller.rb]: 👍

**Review by martinjaimem — APPROVED** [review]

**Pull request merged**